src/main/java/net/minecraftforge/common/util/EnumHelper.java

```diff
@@ -141,6 +143,18 @@ public static EnumRarity addRarity(String name, TextFormatting color, String dis
     {
         return addEnum(EnumRarity.class, name, color, displayName);
     }
+    /**
+     * 
+     * @param name - The name the new enum constant should have
+     * @param textureLocation - The path to the texture for this armor type. Formatted as domain:path.
```

Comment:
What path is the `path` relative to? Does `a:b` mean `assets/a/textures/b.png`? `assets/a/textures/horse_armor/b.png`? `assets/a/b.png`?